mlx/directives/item_2d_matrix_directive.py

```diff
@@ -97,11 +97,11 @@ def run(self):
         self.process_title(node, '2D traceability matrix of items')
 
         self.process_options(node,
-                             {'target': '',
-                              'source': '',
-                              'type': [],
-                              'hit': 'x',
-                              'miss': '',
+                             {'target': {'default': ''},
+                              'source': {'default': ''},
+                              'type':   {'default': []},
+                              'hit':    {'default': 'x'},
+                              'miss':   {'default': ''},
```

Comment:
Not a fan of alignment like this because it's not as easy to maintain.

Reply:
See e5dc5dc and 5d5ba9c